index.yaml

```diff
@@ -31268,1832 +31268,6 @@ entries:
-    dependencies:
-    - name: common-library
-      repository: https://helm-charts.newrelic.com
-      version: 1.1.1
-    description: A Helm chart to install New Relic Super agent on Kubernetes
-    digest: c10fdef58cf2c521a793f63696464523d697e91e721909cd4b8e0bdbf4a66291
-    keywords:
-    - newrelic
-    - super-agent
-    maintainers:
-    - name: sigilioso
-      url: https://github.com/sigilioso
-    - name: gsanchezgavier
-      url: https://github.com/gsanchezgavier
-    - name: kang-makes
-      url: https://github.com/kang-makes
-    - name: marcsanmi
-      url: https://github.com/marcsanmi
-    - name: paologallinaharbur
-      url: https://github.com/paologallinaharbur
-    name: super-agent-deployment
-    type: application
-    urls:
-    - https://github.com/newrelic/helm-charts/releases/download/super-agent-deployment-0.0.2-beta/super-agent-deployment-0.0.2-beta.tgz
-    version: 0.0.2-beta
-  - apiVersion: v2
-    appVersion: nightly
-    created: "2023-11-16T15:48:24.26662433Z"
-    dependencies:
-    - name: common-library
-      repository: https://helm-charts.newrelic.com
-      version: 1.1.1
-    description: A Helm chart to install New Relic Super agent on Kubernetes
-    digest: 5635ee3a61ef5821a0fb9fa1500e5e1f77c106b92ddbe0dfcf8e75bbd18d9cfa
-    keywords:
-    - newrelic
-    - super-agent
-    maintainers:
-    - name: sigilioso
-      url: https://github.com/sigilioso
-    - name: gsanchezgavier
-      url: https://github.com/gsanchezgavier
-    - name: kang-makes
-      url: https://github.com/kang-makes
-    - name: marcsanmi
-      url: https://github.com/marcsanmi
-    - name: paologallinaharbur
-      url: https://github.com/paologallinaharbur
-    name: super-agent-deployment
-    type: application
-    urls:
-    - https://github.com/newrelic/helm-charts/releases/download/super-agent-deployment-0.0.1-beta/super-agent-deployment-0.0.1-beta.tgz
-    version: 0.0.1-beta
-  - apiVersion: v2
-    appVersion: nightly
-    created: "2023-10-23T15:27:35.32049743Z"
-    dependencies:
-    - name: common-library
-      repository: https://helm-charts.newrelic.com
-      version: 1.1.1
-    description: A Helm chart to install New Relic Super agent on Kubernetes
-    digest: b9acfd972adfc73e9e4065f876eb970cc6547aac80f21e5829f4a88778b18891
-    keywords:
-    - newrelic
-    - super-agent
-    maintainers:
-    - name: sigilioso
-      url: https://github.com/sigilioso
-    - name: gsanchezgavier
-      url: https://github.com/gsanchezgavier
-    - name: kang-makes
-      url: https://github.com/kang-makes
-    - name: marcsanmi
-      url: https://github.com/marcsanmi
-    - name: paologallinaharbur
-      url: https://github.com/paologallinaharbur
-    name: super-agent-deployment
-    type: application
-    urls:
-    - https://github.com/newrelic/helm-charts/releases/download/super-agent-deployment-0.0.0-beta/super-agent-deployment-0.0.0-beta.tgz
-    version: 0.0.0-beta
```

Comment:
I would update the last line of the file just in case. 
Moreover, you can test the index before merging it! Have you done it? there is a workflow in this repo showing how to add it pointing to a branch